### PR TITLE
Remove duplicate product submenu

### PR DESCRIPTION
**Pull request by Sinetheta**
I have a feeling we're going to be adding this to a lot of gems as long as we want to support old solidus.

### CODE REVIEW

File: app/views/spree/admin/product_packages/index.html.erb [hunk 1]
```diff
@@ -6,7 +6,8 @@
   <%= javascript_include_tag 'admin/product_packages/index.js' %>
 <% end %>
 
-<%= render :partial => 'spree/admin/shared/product_sub_menu' %>
+<%# This partial was remove in solidus 1.2 but that is also when Spree.solidus_version was added %>
+<%= render :partial => 'spree/admin/shared/product_sub_menu' if !Spree.respond_to?(:solidus_version) %><%= render :partial => 'spree/admin/shared/product_tabs', :locals => { :current => 'Product Packages' } %>
```
Comment (fredericboivin) [inline app/views/spree/admin/product_packages/index.html.erb]: I think the last part of the line (render product_tabs) should not be there (it's already present below so this is a duplicate)

Comment (Sinetheta) [inline app/views/spree/admin/product_packages/index.html.erb]: Thanks @dangerdogz I hate long lines, this was me copy pasting between gems.

**Comment by fredericboivin** [comment]
Pulled this down locally because I wasn't quite sure about what it was supposed to fix (now I see)  

+1

**Comment by Sinetheta** [comment]
Ya, I make changes to core so that I can then fix them in gems and feel smart 😉

**Comment by forkata** [comment]
Thanks @Sinetheta, always appreciate you fixing things you broke ;)

**Pull request merged**